README.md

```diff
@@ -51,8 +51,8 @@ on the snap, including how to install it, please refer to [EdgeX snap](https://g
 
 ##### Go
 
-- The current targeted version of the Go language runtime for release artifacts is v1.13.x
-- The minimum supported version of the Go language runtime is v1.13.x (currently v1.13)
+- The current targeted version of the Go language runtime for release artifacts is v1.15.x
+- The minimum supported version of the Go language runtime is v1.13.x
```

Comment:
This should be 1.15.x since we are forcing that in the go.mod files.

Reply:
I think the readme we have always suggested min version and my thought was 1.13 was still ok to use.  I change it though.

Reply:
The version we put in the go.mod file specifies the min version, so in our case it will be 1.15.x